test/controllers/stripe_connect_events_controller_test.exs

```diff
@@ -54,9 +54,7 @@ defmodule CodeCorps.StripeConnectEventsControllerTest do
 
       assert response(conn, 200)
 
-      updated_account =
-        StripeConnectAccount
-        |> Repo.get_by(id_from_stripe: stripe_id)
+      updated_account = StripeConnectAccount |> Repo.get_by(id_from_stripe: stripe_id)
```

Comment:
Thoughts on not piping here? `updated_account = Repo.get_by(StripeConnectAccount, id_from_stripe: stripe_id)`